sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -994,18 +999,21 @@ class SGDClassifier(BaseSGDClassifier):
         important to get the prediction exactly right.
         For epsilon-insensitive, any differences between the current prediction
         and the correct label are ignored if they are less than this threshold.
+        Values should be in the range `[0.0, inf)`.
 
     n_jobs : int, default=None
         The number of CPUs to use to do the OVA (One Versus All, for
         multi-class problems) computation.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
+        Values should be in the range `[-1, inf)`.
```

Comment:
They can be any integers. I don't think we should document the range here. The Glossary is enough for `n_jobs`:

```suggestion
```